src/Form/CitizenProjectCommandType.php

```diff
@@ -133,6 +138,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->get('skills')->addModelTransformer($this->citizenProjectSkillTransformer);
         $builder->get('committees')->addModelTransformer($this->committeeTransformer);
         $builder->get('address')->remove('address');
+
+        // Removes the "remove_image" field on the creation form or if the image was not upload
+        if (!$citizenProject instanceof CitizenProject || !$citizenProject->hasImageUploaded()) {
```

Comment:
Let's invert the logic and only add the field if the CP has an image, what do you think ?